### PR TITLE
Review and Update SFTP Mode

### LINKED ISSUE #5472: Developing on Pantheon Directly with SFTP Mode Doc Update
Re: [Developing on Pantheon Directly with SFTP Mode](https://pantheon.io/docs/sftp)

Priority: Low

## Issue Description:
The term "Binding path" isn't immediately clear to customers who are trying to find their code, so the header "How do I find my site's Binding path?" doesn't help when their SFTP client doesn't open the right directory.

## Suggested Resolution
What if we changed the heading to something like "I can't find my site code directory" or something similar?

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #5472

## Effect
PR includes the following changes:
- Review and update throughout.
- Improved language around remote path FAQ answer.

## Remaining Work
- [x] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/sftp.md [hunk 1]
```diff
@@ -101,77 +106,83 @@ SFTP mode works with any standards-compliant SFTP client, including many GUI too
...
 </Alert>
 
-You can find the Remote Path after [connecting to SFTP](#sftp-connection-information) via command line, using the `pwd` command:
+You can find the Remote Path, also known as the **binding path**, after [connecting to SFTP](#sftp-connection-information) via command line, using the `pwd` (print working directory) command:
 
-```
+```bash
 sftp> pwd
 Response: Remote working directory: /srv/bindings/daa068ccf4f8414596cddf5xxxxx
 ```
```
Comment (alexfornuto) [inline source/content/sftp.md]: @sparklingrobots this is the section of changes that closes #5472, humbly submitted for your review.

Comment (sparklingrobots) [reply source/content/sftp.md]: This looks great; no changes.  Thank you! 👍

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5526 | Difference |
| --- | --- | --- | --- |
| Performance Score | `60` | `40` | `-20` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5526` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18500/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18500/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5526 | Difference |
| --- | --- | --- | --- |
| Performance Score | `79` | `47` | `-32` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `96` | `-2` |

Lighthouse Audit Reports:

- [`pr-5526` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18506/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18506/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**